updates to existing entries, mostly home urls, code repos and license

Trilarion opened the pull request:
Advanced Strategic Command
- repo is offline for some time, maintainer exported the Mercurial repository recently to Github

Ancient Beast
- home redirects to lowercase URL version

Antares
- code license is LGPL3 (https://arescentral.org/antares: "The latest version of Antares is 0.9.0, released 15 October, 2018. It is licensed under the GNU Lesser General Public License 3.0.")

Astromenace
- code has moved to Github (https://github.com/viewizard/astromenace)

BZFlag
- home redirects to https://www.bzflag.org/ (minor change)

Crimson Fields
- development has halted since many years

Dune Dynasty
- development has halted (no activity since 2016)

Exult
- repo moved to github and there is recent, activity (see https://github.com/exult/exult/graphs/contributors)

FlightGear
- home redirects to https://www.flightgear.org/ (minor change)

Freeciv
- repo moved to github, there is lots of activity (active -> very active)

KAtomic
- repo link outdated, is now at https://cgit.kde.org/katomic.git/

LGeneral
- home link outdatd, is now http://lgames.sourceforge.net/LGeneral/

micropolisJS
- sporadic development, there was a bit of activity in 2018 (see https://github.com/graememcc/micropolisJS/graphs/contributors)

Minetest
- license of source code is LGPL2 (see https://github.com/minetest/minetest/blob/master/LICENSE.txt)

NLarn
- old home url redirects, repo is now on github, license has been upgraded to GPL3

Open Surge
- newer home page, repo moved to github, license has been upgraded to GPL3

OpenArena
- home page redirect, repo moved to github

OpenClonk
- old repo url redirected to github url (replaced with redirect)

OpenLieroX
- license is LGPL2 (see https://github.com/albertz/openlierox/blob/0.59/COPYING.LIB), sporadic development

Rocks'n'Diamonds
- repo url slightly changed

SuperTuxKart
- home url redirects, replaced with redirect

The Battle for Wesnoth
- repo moved to github

TVTower
- repo moved within github

UFO2000
- repo git mirror of svn exists on github, use this

UFO: Alien Invasion
- home url redirects, replace with redirect

Unvanquished
- code license now GPL3 (see https://github.com/Unvanquished/Unvanquished/blob/master/GPL.txt)

XArchon
- code license is GPL2 (see file COPYING in the linked source)

Xonotic
- repo is on gitlab now and license has been upgraded to GPL3 (see https://gitlab.com/xonotic/xonotic/blob/master/COPYING)

cxong commented on games/f.yaml:
```diff
@@ -326,11 +326,12 @@
   license:
   - GPL2
-  development: active
+  development: very active
```
```suggestion
  development: active
```
Last commit 24 days ago; not very active

Trilarion replied:
Done

cxong commented on games/r.yaml:
```diff
@@ -431,10 +431,10 @@
   - Boulder Dash
   - Supaplex
-  repo: http://git.artsoft.org/?p=rocksndiamonds.git
+  repo: http://git.artsoft.org/rocksndiamonds.git
```
```suggestion
  repo: https://git.artsoft.org/rocksndiamonds.git/
```

Trilarion replied:
Done

cxong commented on games/o.yaml:
```diff
@@ -387,9 +387,10 @@
   type: remake
-  url: http://openarena.ws/
+  updated: 2019-08-20
+  url: http://openarena.ws/smfnews.php
```
```suggestion
  url: http://openarena.ws/
```
I would leave this unchanged because if they do change their web framework, the url will still be ok

Trilarion replied:
Done

cxong commented on games/s.yaml:
```diff
@@ -1154,8 +1154,8 @@
-  updated: 2014-05-08
-  url: http://supertuxkart.sourceforge.net/
+  updated: 2019-08-20
+  url: https://supertuxkart.net/Main_Page
```
```suggestion
  url: https://supertuxkart.net/
```

Trilarion replied:
Done

cxong commented on games/u.yaml:
```diff
@@ -50,10 +50,10 @@
   - 'X-COM: Apocalypse'
   - 'UFO: Enemy Unknown'
-  repo: https://sourceforge.net/projects/ufo2000/
+  repo: https://github.com/ufo2000/ufo2000
```
```suggestion
  repo: https://sourceforge.net/projects/ufo2000/
```
I don't think this should be changed; the github description says it is a repo mirror. The project download page still links to sourceforge

Trilarion replied:
Done

Trilarion commented:
I did all the changes that were requested, but then I didn't want to update the "update dates", rebased my fork on the actual master here and pushed the single commit with force to my branch. The automatic script still complained about the update dates, so I updated them.  The pull request is now the original request with all the requested changes and the update date of today.

Trilarion commented:
Unfortunately, the changes requested overview has not been updated accordingly.

cxong commented:
FYI the updated dates message is only a warning; it doesn't need to be strictly followed
